mock/service/controller.go

```diff
@@ -341,3 +341,23 @@ func (s *service) ControllerGetCapabilities(
 		},
 	}, nil
 }
+
+func (s *service) CreateSnapshot(ctx context.Context,
+	req *csi.CreateSnapshotRequest) (*csi.CreateSnapshotResponse, error) {
+	return &csi.CreateSnapshotResponse{}, nil
+}
+
+func (s *service) DeleteSnapshot(ctx context.Context,
+	req *csi.DeleteSnapshotRequest) (*csi.DeleteSnapshotResponse, error) {
+	return &csi.DeleteSnapshotResponse{}, nil
+}
+
+func (s *service) ListSnapshots(ctx context.Context,
+	req *csi.ListSnapshotsRequest) (*csi.ListSnapshotsResponse, error) {
+	return &csi.ListSnapshotsResponse{}, nil
+}
+
+func (s *service) NodeGetInfo(ctx context.Context,
```

Comment:
`NodeGetInfo` should be part of node service not controller

Reply:
copy and past thing, thanks for catching..